pkgs/by-name/sc/screenconfig/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrandr,
+  srandrd,
+  feh,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "screenconfig";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jceb";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    repo = "screenconfig";
    tag = "v${version}";
```
#277994
Also please squash all commits. Then I can merge.